aea/manager.py

```diff
@@ -211,25 +251,31 @@ def add_error_callback(
         """Add error callback to call on error raised."""
         self._error_callbacks.append(error_callback)
 
-    def start_manager(self) -> "MultiAgentManager":
+    def start_manager(self, local: bool = True) -> "MultiAgentManager":
         """Start manager."""
         if self._is_running:
             return self
 
         self._ensure_working_dir()
+        self._load_state(local=local)
         self._started_event.clear()
         self._is_running = True
         self._thread = Thread(target=self._run_thread, daemon=True)
         self._thread.start()
         self._started_event.wait(self.DEFAULT_TIMEOUT_FOR_BLOCKING_OPERATIONS)
         return self
 
-    def stop_manager(self) -> "MultiAgentManager":
+    def stop_manager(
+        self, cleanup: bool = True, save: bool = False
+    ) -> "MultiAgentManager":
```

Comment:
booleans should be named with `is_`,`has_` or `can_`

Reply:
There're a lot of bool params here named `local` or `cleanup`. IMO `save` option sounds consistent on this case.